src/main/java/br/com/alura/comex/controller/AutenticacaoController.java

```diff
@@ -0,0 +1,40 @@
+package br.com.alura.comex.controller;
+
+import br.com.alura.comex.configuration.security.service.TokenService;
+import br.com.alura.comex.configuration.security.form.LoginForm;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/auth")
+public class AutenticacaoController {
+
+    @Autowired
+    private AuthenticationManager authManager;
+
+    @Autowired
+    private TokenService tokenService;
+
+    @PostMapping
+    public ResponseEntity<br.com.alura.comex.configuration.security.model.Token> autenticar(@RequestBody @Valid LoginForm form) {
```

Comment:
Daria pra colocar o import e evitar esse textão todo!

```suggestion
    public ResponseEntity<Token> autenticar(@RequestBody @Valid LoginForm form) {
```